RISE-V2G-SECC/src/main/java/com/v2gclarity/risev2g/secc/states/WaitForPowerDeliveryReq.java

```diff
@@ -152,8 +152,11 @@ public boolean isResponseCodeOK(PowerDeliveryReqType powerDeliveryReq) {
 				dcEVSEStatusCode.equals(DCEVSEStatusCodeType.EVSE_EMERGENCY_SHUTDOWN) || 
 				dcEVSEStatusCode.equals(DCEVSEStatusCodeType.EVSE_MALFUNCTION)) {
 				getLogger().error("EVSE status code is '" + dcEVSEStatusCode.toString() + "'");
```

Comment:
In this case, the logging should probably better be a warning instead of an error.